### DISCUSSION TITLE
allowedVersions not working for node engine version

### DISCUSSION THREAD

**Discussion by The-Code-Monkey** [renovatebot/renovate#15472] [Request Help] [answered]
### How are you running Renovate?

WhiteSource Renovate hosted app on github.com

### Was this something which used to work for you, and then stopped?

I never saw this working

### Describe the bug

```
"packageRules": [
    {
      "matchPackageNames": ["node"],
      "allowedVersions": "< 17.0.0"
    }
  ],
```

this is the rule im running but its trying to use v18

linked repo

https://github.com/The-Code-Monkey/TechStack/blob/dev/renovate.json

### Relevant debug logs

https://gist.github.com/The-Code-Monkey/5a27bf50d60ea945457b0d6f2075bea1

### Have you created a minimal reproduction repository?

No reproduction, but I have linked to a public repo where it occurs

**Comment by The-Code-Monkey** [comment]
Reason i need this to work, is that i build on vercel which supports 16 at max.

**Comment by rarkins** [comment, marked as answer]
`allowedVersions` is only applied to upgrades, while you are getting a Pin Dependencies PR which pins node to the highest version matching of your current range.

You gave your engines an open ended range, example: https://github.com/The-Code-Monkey/TechStack/blob/ad7a522fd16ba26f26c5c0ee4f03b37887e07b26/packages/styled-system/package.json#L26-L28

**Reply by The-Code-Monkey** [reply]
I understand i have an open ended range but, the packageRules should override what the engines value says in package json to me, or at least it should be clearer what the docs say it does, otherwise i have to go in and manually edit each package json, might as well not use renovate at that point.

**Reply by rarkins** [reply]
PRs to the docs would be welcome, you're also welcome to stop using Renovate in protest if you wish
